init/x86_64/Ubuntu_16.04/shippable-reqKick.env.template

```diff
@@ -1,3 +1,4 @@
 STATUS_DIR="{{STATUS_DIR}}"
 SCRIPTS_DIR="{{SCRIPTS_DIR}}"
 REQEXEC_BIN_PATH="{{REQEXEC_BIN_PATH}}"
+RUN_MODE="{{RUN_MODE}}"
```

Comment:
You need this in the init script for aarch as well. Will that be in a separate PR.